### ISSUE TITLE
[1.14.6] Can't handle huge playlists since d4567ee

### ISSUE THREAD

**Issue by tmn505** [pupnp/pupnp#330]
The issue is observed when a huge playlist is feeded to VLC through UPNP. The VLC outputs these messages:
```
upnp info: Initializing libupnp on 'default' interface
upnp error: No response from browse() action
```
This issue does not occur before d4567ee.
The playlist causing the issue is attached: [channels.m3u.txt](https://github.com/pupnp/pupnp/files/6785624/channels.m3u.txt)

**Comment by tmn505** [comment]
Fixed with 1.14.13, thanks. Didn't bisect which commit did it, if it'll be requested, I'll do the bisection.

**Comment by mrjimenez** [comment]
If you want to do it, it will be good so that we do not reintroduce the regression by mistake.

Regards,
Marcelo.

**Comment by AlaricSenat** [comment]
Looks like the symptoms fixed by #406 !
